src/controlCenter/widgets/buttonsGrid/buttonsGrid.vala

```diff
@@ -15,15 +15,27 @@ namespace SwayNotificationCenter.Widgets {
         public ButtonsGrid (string suffix, SwayncDaemon swaync_daemon, NotiDaemon noti_daemon) {
             base (suffix, swaync_daemon, noti_daemon);
 
+            Gtk.FlowBox container = new Gtk.FlowBox ();
+            container.set_selection_mode (Gtk.SelectionMode.NONE);
+            pack_start (container, true, true, 0);
+
             Json.Object ? config = get_config (this);
             if (config != null) {
                 Json.Array a = get_prop_array (config, "actions");
                 if (a != null) actions = parse_actions (a);
-            }
 
-            Gtk.FlowBox container = new Gtk.FlowBox ();
-            container.set_selection_mode (Gtk.SelectionMode.NONE);
-            pack_start (container, true, true, 0);
+                bool? center = get_prop<bool> (config, "center");
+                if (center != null && center)
+                    container.set_halign(Gtk.Align.CENTER);
+
+                int? col_min = get_prop<int> (config, "column-min");
+                if (col_min != null && col_min > 0)
+                    container.set_min_children_per_line(col_min);
+
+                int? col_max = get_prop<int> (config, "column-max");
+                if (col_max != null && col_max > 0)
+                    container.set_max_children_per_line(col_max);
```

Comment:
The linter requires spaces between the function name and the parenthesis. Also, I'd prefer if you use brackets on small if-statments.

Not sure why GitHub isn't allowing me to run my GitHub Linting Actions…

Reply:
The vala dev settings on my laptop not seems to be stable enough, reformat with uncrustify, feel free to adjust the code on merge if you like ;)